__tests__/news.test.js

```diff
@@ -86,4 +86,26 @@ describe("GET", () => {
         });
       });
   });
+
+  test("/api/articles/:article_Id", () => {
+    return request(app)
+      .patch("/api/articles/5")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then((res) => {
+        const article_5 = {
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          body: "Bastet walks amongst us, and the cats are taking arms!",
+          created_at: "2020-08-03T13:14:00.000Z",
+          votes: 1,
+        };
+
+        const articleObj = res.body.article;
+
+        expect(articleObj).toEqual(article_5);
+      });
+  });
 });
```

Comment:
Theres a few more tests id like to see here, specifically around error handling. 
What would happen if i tried to patch article 99999 (out of range id)
What would happen if i tried to patch article 'dog' (incorrect data type for the id)
what would happen if i sent a body of {inc_votes: 'ten'} (incorrect data type for the vote)
what would happen if i sent a body of {votes: 1} (incorrect key)